### PR TITLE
Possible fix for #1398

### PR DESCRIPTION
**Pull request by karanjthakkar**

### CODE REVIEW

File: src/tooltip/tooltip.js [hunk 1]
```diff
@@ -133,10 +133,12 @@
         }
 
         this.boundMouseEnterHandler = this.handleMouseEnter_.bind(this);
-        this.boundMouseLeaveHandler = this.handleMouseLeave_.bind(this);
+        this.boundMouseLeaveHandler = this.hideTooltip_.bind(this);
+        this.boundScrollHandler = this.hideTooltip_.bind(this);
```
Comment (Garbee) [inline src/tooltip/tooltip.js]: You don't need to bind the same thing twice. Simply do one bound handler and re-use it for each event listener. Same result, less memory.

Comment (karanjthakkar) [inline src/tooltip/tooltip.js]: @Garbee Oh yes. Fixed.

**Comment by karanjthakkar** [comment]
@Garbee Any clue why these tests are failing? Not related to my change actually.

**Comment by Garbee** [comment]
This error is related: `this.forElement_.addEventListener('mouseleave', this.boundMouseLeaveHandler, false);`

Other than that, just ignore them. I'll fix it in a minute in the branch directly. Once that is done I'll do a merge locally and test to be sure before pushing the patch in.

**Comment by karanjthakkar** [comment]
@Garbee Fantastic! 👍

**Comment by karanjthakkar** [comment]
@Garbee Bump.

**Comment by karanjthakkar** [comment]
@Garbee Everything seems to be working now. What do you think?

**Comment by Garbee** [comment]
Squashed and committed with 5065fe8d41c2bb233956f240ff7620f59ba2c00a.

Thank you!

**Pull request closed without merging**